app/segments/drill/[id]/submission/input.js

```diff
@@ -224,12 +341,15 @@ function createOutputData(
     }
 
     //add the sid to the shot
-    shot.sid = j;
+    shot.sid = j + 1;
```

Comment:
There's 2 of them???

Reply:
There was some whack stuff that happened when I rebased. I tried to go in a fix it but must've missed this one, good catch.